eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -18,6 +18,13 @@
 abstract class RoleCreateStruct extends ValueObject
 {
     /**
+     * ID of a role.
+     *
+     * @var int
+     */
+    public $id;
```

Comment:
Well, in that case, why not call it existingRoleId, or similar ? The reason is good, the name: not that much :-)

Would that work ?